src/lib/swapper/swappers/LifiSwapper/utils/getNetworkFeeCryptoBaseUnit/getNetworkFeeCryptoBaseUnit.ts

```diff
@@ -54,11 +55,15 @@ export const getNetworkFeeCryptoBaseUnit = async ({
         type: 'function',
       },
     ]
-    const web3 = getWeb3InstanceByChainId(chainId)
-    const tokenContract = new web3.eth.Contract(optimismGasOracleAbi, OPTIMISM_GAS_ORACLE_ADDRESS)
-    const l1Fee = await tokenContract.methods.getL1Fee(data).call()
 
-    return gasFee.plus(l1Fee).toString()
+    const tokenContract = new ethers.Contract(
+      OPTIMISM_GAS_ORACLE_ADDRESS,
+      optimismGasOracleAbi,
+      getEthersProvider(chainId),
+    )
+
+    const l1Fee = (await tokenContract.getL1Fee(data)) as BigNumber
```

Comment:
Now that lib is in web, and that this is now an ethers contract, we should be able to add the abi in `contractManager` and get typechain types for this so we can remove the cast, but let's not tackle in this PR

Reply:
Good point, also contemplated exposing this in unchained since we are already doing it there as well. I wish the node just had a whitelist address to call estimate gas that acts like it has unlimited funds so we can always do full node estimations even on quote haha